pkg/reconciler/pipelinerun/resources/pipelinerunresolution_test.go

```diff
@@ -2836,21 +2837,27 @@ func TestGetTaskRunNames(t *testing.T) {
 		ptName: "task2-0123456789-0123456789-0123456789-0123456789-0123456789",
 		prName: "pipeline-run-0123456789-0123456789-0123456789-0123456789",
 		wantTrNames: []string{
-			"pipeline-run-0123456789-01234569d54677e88e96776942290e00b578ca5",
 			"pipeline-run-0123456789-01234563c0313c59d28c85a2c2b3fd3b17a9514",
+			"pipeline-run-0123456789-01234569d54677e88e96776942290e00b578ca5",
 		},
 	}} {
 		t.Run(tc.name, func(t *testing.T) {
 			testPrName := prName
 			if tc.prName != "" {
 				testPrName = tc.prName
 			}
-			trNameFromTRStatus := GetNamesofTaskRuns(taskRunsStatus, nil, tc.ptName, testPrName, 2)
-			if d := cmp.Diff(tc.wantTrNames, trNameFromTRStatus); d != "" {
+			namesOfTaskRunsFromTaskRunsStatus := GetNamesOfTaskRuns(taskRunsStatus, nil, tc.ptName, testPrName, 2)
+			sort.Slice(namesOfTaskRunsFromTaskRunsStatus, func(i, j int) bool {
+				return namesOfTaskRunsFromTaskRunsStatus[i] < namesOfTaskRunsFromTaskRunsStatus[j]
+			})
+			if d := cmp.Diff(tc.wantTrNames, namesOfTaskRunsFromTaskRunsStatus); d != "" {
```

Comment:
```
if d := cmp.Diff(sort.Strings(tc.wantTrNames), sort.Strings(namesOfTaskRunsFromTaskRunsStatus)); d != "" {
```

Reply:
haha had followed recommendation from here because I was looking for cmp.IgnoreOrder: https://github.com/google/go-cmp/pull/71#issuecomment-501669949

also, the sort function does not return a slice so can't do it inline